src/main/webapp/file-download-popup-fragment.xhtml

```diff
@@ -43,18 +44,21 @@
                     </div>
                 </div>
             </div>
-            <ui:fragment rendered="#{workingVersion.dataset.guestbook != null and workingVersion.dataset.guestbook.enabled and downloadPopupRequired}" id="guestbookUIFragment">
+            <p:fragment rendered="#{workingVersion.dataset.guestbook != null and workingVersion.dataset.guestbook.enabled and downloadPopupRequired}" id="guestbookUIFragment">
                 <p:fragment>
                     <div class="form-group">
                         <label class="col-sm-3 control-label" for="guestbookuser_nameText">
                             #{bundle.name}
                             <span class="glyphicon glyphicon-asterisk text-danger" jsf:rendered="#{workingVersion.dataset.guestbook.nameRequired}" />
```

Comment:
Somewhere in this xhtml file we should add an extra space for consistency. The non-custom questions have a space before the red asterisk: 

![screen shot 2018-01-11 at 12 12 41 pm](https://user-images.githubusercontent.com/21006/34838219-53bdeea4-f6cb-11e7-9617-afc4b62b50f8.png)

Reply:
Suggested changes/fix have been made.